tests/acceptance/lib.ts

```diff
@@ -53,7 +53,20 @@ if (Meteor.isServer) {
       currentTest = testName;
       currentConn = this.connection;
 
-      xolvioResetDatabase();
+      // Remove all the contents of all not-mongo-internal collections
+      const excludedCollections = ['system.indexes'];
+      const db = MongoInternals.defaultRemoteCollectionDriver().mongo.db;
+      const collections = await db.collections();
+      const appCollections = collections.filter((col) => {
+        // Exclude velocity collections and system indexes.
+        return col.collectionName.indexOf('velocity') !== 0 && excludedCollections.indexOf(col.collectionName) === -1;
```

Comment:
I'd be slightly more comfortable if this was `col.collectionName.startsWith('system.')`

Reply:
Can do.  This was just the existing `xolvio:cleaner` behavior.